### ISSUE TITLE
Create Home Page tests

### ISSUE THREAD

**Issue by mr-ryan12** [mr-ryan12/rancid-tomatillos#17]
- [x] Should be able to visit the main page and the user will see the application name & fun tagline
- [x] Should display the poster image passed
- [x] Should display the movie title
- [x] Should have an alt text for the image
- [x] Should display the release date
- [x] Should have a clickable poster image
- [x] Should display the average rating
- [x] Should not display the backdrop image
- [x] Should display 3 movie cards
- [x] Should display the movies sorted by release date (newest to oldest)
